src/TCommandLine.cpp

```diff
@@ -1288,3 +1293,94 @@ void TCommandLine::clearSuggestions()
 {
     commandLineSuggestions.clear();
 }
+
+void TCommandLine::slot_adjustAccessibleNames()
+{
+    bool isMultipleProfilesActive = (mudlet::self()->getHostManager().getHostCount() > 1);
+    const QString hostName{mpHost ? mpHost->getName() : QString()};
+    switch (mType) {
+    case MainCommandLine:
+        if (isMultipleProfilesActive) {
```

Comment:
Can we go with `if (multipleProfilesActive)`? I find it much more readable.

Reply:
I tend to use an `*`is`* prefix for booleans and I don't recall anything in our coding style about it. Let me see if I can cleanly change this here (and in the other PR that now incorporates it)...

Reply:
Done.